alternator/rjson.hh

```diff
@@ -162,6 +169,14 @@ void set_with_string_name(rjson::value& base, std::string_view name, rjson::stri
 // Throws if base is not a JSON object.
 void set(rjson::value& base, rjson::string_ref_type name, rjson::value&& member);
 
+template<typename T>
+std::enable_if_t<!std::is_constructible_v<string_ref_type, T>> 
```

Comment:
This is to disambiguate from the set(..., string_ref_type) method. Otherwise we get multi match on overload. So if you already have a string, you go to the string. If you have an non-object  (UUID), you go here.